apollo-router/src/plugins/include_subgraph_errors.rs

```diff
@@ -73,6 +79,33 @@ impl Plugin for IncludeSubgraphErrors {
         }
         service
     }
+
+    fn execution_service(&self, service: execution::BoxService) -> execution::BoxService {
```

Comment:
it sure would be nice to do this in a fetch_service hook, huh! maybe add a todo comment suggesting that?